syft/pkg/cataloger/javascript/package.go

```diff
@@ -91,6 +92,7 @@ func newPackageLockV2Package(resolver source.FileResolver, location source.Locat
 			Language:  pkg.JavaScript,
 			Type:      pkg.NpmPkg,
 			Licenses:  licenses,
+			Metadata:  Metadata{Resolved: u.Resolved},
```

Comment:
There are a few more changes I think we need here:
This should be named something more unique like `NpmMetadata`, and we also need to set a corresponding `MetadataType` in the package struct here. These currently all defined in: [syft/pkg](https://github.com/anchore/syft/blob/main/syft/pkg/)

We also need to add any new metadata types to the appropriate spots in [syft/pkg/metadata.go](https://github.com/anchore/syft/blob/main/syft/pkg/metadata.go)

Similar comments for the `python.Metadata` below.

Reply:
done

Reply:
the latest commit accounts for this (see https://github.com/anchore/syft/pull/1327#issuecomment-1380925363 and https://github.com/anchore/syft/pull/1327/commits/8cecacfbfd6fc6c16be985cbbecdffffb1dd6884)